ziti/cmd/ziti/cmd/config_templates/controller.yml

```diff
@@ -27,7 +27,7 @@ ctrl:
   #connectTimeoutMs:       {{ .Controller.DefaultConnectTimeout.Milliseconds }}
   listener:             tls:{{ .Controller.ListenerAddress }}:{{ .Controller.Port }}
 
-# the endpoint that management tools connect to the controller over.
+# endpoint that can be used to for some streaming operations, such as streaming circuit events and metrics (see 'web' for Edge mgmt REST API)
```

Comment:
There seems to be a grammar issue here. ("can be used to for"), I think the "to" should be removed.